lib/cache/cache.go

```diff
@@ -107,6 +109,7 @@ func ForRemoteProxy(cfg Config) Config {
 		{Kind: services.KindAppServer},
 		{Kind: services.KindRemoteCluster},
 		{Kind: services.KindKubeService},
+		{Kind: services.KindDatabaseServer},
```

Comment:
This is not part of your changes, but can you update `ForOldRemoteProxy` below with the list of resources from 5.0 and then update the comment to bump it to delete in the next version.

Reply:
Done.